### ISSUE TITLE
Calling np.sum failed in Phylanx

### ISSUE THREAD

**Issue by weilewei** [STEllAR-GROUP/phylanx#489]
calling np.sum will raise complains, but sum works fine.

testing code:
```
def python_np_sum(input):
    print(np.sum(input))


@Phylanx(debug=True)
def phylanx_sum(a):
    print(sum(a))


@Phylanx(debug=True)
def phylanx_np_sum(a):
    print(np.sum(a))


a = np.array([1, 2, 3])

python_np_sum(a)
phylanx_sum(a)
phylanx_np_sum(a) # it will complain as np.sum does not work in phylanx
```

Can anyone take a look at this? Very appreciate.

**Comment by hkaiser** [comment]
@rtohid is that something that needs to be added to the list of mapped primitives?

**Comment by rtohid** [comment]
@hkaiser I don't think so. Since the keyword `sum` is the same in Phylanx and NumPy, `np.sum` should automatically get mapped to Phylanx's `sum` primitive.

@weilewei Could you please verify that other NumPy methods are mapped correctly? As I remember you also used `np.dot` and `np.transpose` in your code, thanks.

**Comment by rtohid** [comment]
@hkaiser Sorry, you are right, I had forgotten that we still explicitly map NumPy methods! I'll fix it.

**Comment by hkaiser** [comment]
This was fixed by merging #526
